sqlcompyre/results/names.py

```diff
@@ -39,12 +50,22 @@ def in_common(self) -> list[str]:
     @cached_property
     def missing_left(self) -> list[str]:
         """Ordered list of names provided only by the "right" database object."""
-        return sorted(self._set_right - self._set_left)
+        if self._name_mapping:
```

Comment:
[nit]

```suggestion
        if self._name_mapping is not None:
```

Reply:
Fine with me, but what's the advantage? The condition will now also be true for empty dictionaries in `self._name_mapping`, right?

Reply:
Yeah, you're right. I like to think this is a little more explicit but yeah, it would also catch the case where the dict is empty. Maybe would be better the way you had it first :smile:

Reply:
Yes, I agree; I reverted it.